AmazonChimeSDKDemo/AmazonChimeSDKDemo.xcodeproj/project.pbxproj

```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 52;
+	objectVersion = 54;
```

Comment:
what caused this update? is it intentional?

Reply:
The objectVersion property in the project.pbxproj file specifies the version of the Xcode project file format. It is auto generated, and I think that is because my Xcode version is 15, but it did not lead to version incompatibility as the integration test build on Xode14 succeeded.